src/main/target/KISSFC/target.h

```diff
@@ -24,6 +24,10 @@
 #define SBUS_PORT_OPTIONS (SERIAL_STOPBITS_2 | SERIAL_PARITY_EVEN | SERIAL_INVERTED | SERIAL_BIDIR)
 
 #define USE_DSHOT
+#define USE_ESC_TELEMETRY
+
+#define USE_ESCSERIAL
+#define ESCSERIAL_TIMER_TX_HARDWARE 6
```

Comment:
Yep, but not related to ESC telemetry tho. It's used for passthrough flashing of KISS ESCs. I had to change this to flash my ESCs with DShot firmware. I can remove it and make a separate pr for it if needed.